### PR TITLE
Rewrote Steppable, Potts, Contact Plugin, & Cell Sorting

### PR DESCRIPTION
**Pull request by fyffep**
I added introductory content for Steppable, Potts, Contact Plugin, & Cell Sorting

### CODE REVIEW

File: docs/contact_plugin.rst [hunk 1]
```diff
@@ -1,28 +1,92 @@
...
+.. math::
+    :nowrap:
+
+    \begin{eqnarray}
+        E_{contact} = \sum_{i,j,neighbors} J\left ( \tau_{\sigma(i)},\tau_{\sigma(j)} \right )\left ( 1-\delta_{\sigma(i), \sigma(j)} \right )
+    \end{eqnarray}
 
+where ``i`` and ``j`` label two neighboring lattice sites :math:`\sigma`'s denote cell
+IDs, and each :math:`\tau` denotes a cell type. 
+Note that ``i`` and ``j`` are are at separate coordinates; for example, ``i = (0,4)`` and ``j = (1,4)``.
```
Comment (fyffep) [inline docs/contact_plugin.rst:86]: I had a hard time understanding this formula, so I added the a few other formulae in this doc. What is delta in this formula? Why do we subtract from 1?
https://pythonscriptingmanual.readthedocs.io/en/latest/contact_plugin.html

File: docs/potts.rst [hunk 2]
```diff
@@ -25,29 +31,42 @@ lattice and the simulation.
...
+   parameter in classical GGH Model formulation. We have decided to use
+   ``FluctuationAmplitude`` term instead of temperature because using the word
+   ``Temperature`` to describe the intrinsic motility of cell membranes was quite
    confusing.
 
 In the above example, fluctuation amplitude applies to all cells in the
 simulation. To define fluctuation amplitude separately for each cell
-type we use the following syntax:
+type, we use the following syntax. 
+Each value must be non-negative.
```
Comment (fyffep) [inline docs/potts.rst:69]: Is the XML below this line for <FluctuationAmplitude> deprecated? I haven't seen it elsewhere.

File: docs/potts.rst [hunk 3]
```diff
@@ -104,8 +116,8 @@ Potts section:
          …
     </Potts>
 
-By default we use ``Min`` function. Notice, that if you use global
-fluctuation amplitude definition ``Temperature`` it does not really matter
+By default, we use the ``Min`` function. Notice that if you use the global
+fluctuation amplitude definition ``Temperature``, it does not really matter
 which function you use. The differences arise when *"source"* and
 *"destination"* cells have different fluctuation amplitudes.
```
Comment (fyffep) [inline docs/potts.rst:122]: I want to delete this and the code below about ``FluctuationAmplitudeFunctionName`` or move it to a different doc. I feel like need two versions of ``potts.rst``: one introductory version and one comprehensive version.

File: docs/contact_plugin.rst (hunk 1 above)
Comment (maciekswat) [reply docs/contact_plugin.rst:86]: this is fine. I like the drawings and simpler explanation

**Review by maciekswat — APPROVED** [review]
This looks great. I made some minor additions but overall it looks very good

**Pull request merged**